dist/ovh-angular-actions-menu.js

```diff
@@ -280,22 +285,29 @@ angular.module("ovh-angular-actions-menu").controller("ActionsMenuCtrl", ["actio
  *      };
  *
  *      $scope.actionsOptions = [{
+ *          id: "firstButton",
  *          title: "My Beautiful title",
  *          icon: "filled-check",
  *          href: "http://www.google.be"
```

Comment:
Obvious Belgian propaganda

Reply:
I also love google Belgium, but maybe we should not chose this as an example